fus.c

```diff
@@ -770,5 +770,11 @@ fus_depsolve (const char *arch,
       g_ptr_array_add (output, name);
     }
 
+  int id;
+  Repo *r;
+  FOR_REPOS(id, r)
+    if (r->appdata)
+      g_free (r->appdata);
```

Comment:
hmm, why do we do that?

Reply:
Because I save the repomd checksum in repo->appdata (repo.c:751)

Reply:
I added a comment to explain why it's needed.